### PR TITLE
Include bulbs-video-play-button and styles

### PR DESCRIPTION
**Pull request by jmelvnsn**
@kand I created this branch from master. Can you let me know what I need to do to make it kosher again?

### CODE REVIEW

File: bulbs/recirc/static/recirc/js/video-recirc.js [hunk 1]
```diff
@@ -21,6 +21,8 @@ VideoRecircList.prototype.videoRecircFetched = function(data) {
 
   that.$videoListTitle.html('More From ' + data.series.name);
 
+  console.log(data.videos);
```
Comment (spra85) [inline bulbs/recirc/static/recirc/js/video-recirc.js]: Remove

File: bulbs/recirc/static/recirc/js/video-recirc.js [hunk 2]
```diff
@@ -32,7 +34,7 @@ VideoRecircList.prototype.videoRecircFetched = function(data) {
       'data-track-label' : videoHref,
       'html': $('<figure>',{
         'class' : 'content',
-        'html' : '<div class="image"><img src="' + posterSource + '/16x9/480.jpg"></div>'
+        'html' : '<div class="image"><bulbs-video-play-button></bulbs-video-play-button><img src="' + posterSource + '/16x9/480.jpg"></div>',
```
Comment (spra85) [inline bulbs/recirc/static/recirc/js/video-recirc.js]: Assume this custom element is registered somewhere else?

**Comment by spra85** [comment]
Other than the few comments, looks 👍

File: bulbs/recirc/static/recirc/js/video-recirc.js (hunk 2 above)
Comment (jmelvnsn) [inline bulbs/recirc/static/recirc/js/video-recirc.js]: Yeah it's in the bulbs-elements PR: https://github.com/theonion/bulbs-elements/pull/1

**Comment by kand** [comment]
👍

**Pull request merged**